### DISCUSSION TITLE
A little help needed. Urgent would be appreciated

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#312] [Q&A]
I'm working on a project with a cool animation, but I'm stuck on a tiny hurdle. I want the animation to seamlessly transition to the "home.js" page once the animation stops.

In other words i want the code of animation when it stops it simply redirects to home.js

Can any of you share the magic code that makes this transition happen?

And remember I WANT FOR ZEPP OS 1.0

**Comment by SashaCX75** [comment]
The animation has a function anim_complete_call. It should be triggered when the animation completes.
https://docs.zepp.com/docs/1.0/reference/device-app-api/hmUI/widget/IMG_ANIM/
In this function, the hmApp.gotoPage command can be used to open any page.
https://docs.zepp.com/docs/1.0/reference/device-app-api/hmApp/gotoPage/
